components/layout/display_list_builder.rs

```diff
@@ -1180,7 +1182,8 @@ impl FragmentDisplayListBuilding for Fragment {
                                 specified(vertical, bounds.size.height)),
             EndingShape::Ellipse(LengthOrPercentageOrKeyword::Keyword(word))
                 => convert_ellipse_size_keyword(word, &bounds.size, &center),
-        };
+        } + Size2D::new((outset.left + outset.right).scale_by(0.5),
```

Comment:
Hmm.. Why does this modify the radius of a radial gradient? Can you quote a spec, or comment on this? I see why it's useful when it's a keyword, or a percentage, but why modifying it when there's an absolute radius?